src/asynk/async_queue.rs

```diff
@@ -383,6 +401,34 @@ mod async_queue_tests {
         assert_eq!(Some("AsyncTask"), type_task);
         transaction.rollback().await.unwrap();
     }
+    #[tokio::test]
+    async fn update_task_state_test() {
+        let pool = pool().await;
+        let mut connection = pool.get().await.unwrap();
+        let mut transaction = connection.transaction().await.unwrap();
+
+        let task =
+            AsyncQueue::<NoTls>::insert_task_query(&mut transaction, &AsyncTask { number: 1 })
+                .await
+                .unwrap();
+        let metadata = task.metadata.as_object().unwrap();
+        let number = metadata["number"].as_u64();
+        let type_task = metadata["type"].as_str();
+        let id = task.id;
+        assert_eq!(Some(1), number);
```

Comment:
just code style comment.

I think it would be more readable if you start separating logical parts in tests with a new line